tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Output.java

```diff
@@ -47,6 +47,24 @@ public Shape shape() {
     return (DataType<T>)operation.dtype(index);
   }
 
+    /**
```

Comment:
The starting comment tag is offset of one character (at least in GitHub review)

Reply:
thanks, fixed!